R/dbConnection.R

```diff
@@ -0,0 +1,111 @@
+#' @import magrittr
+#' @import dplyr
+NULL
+
+#' Make a connection to the Database that can be used for queries
+#'
+#' @export
+connect <- function(host='localhost', username=NULL, password=NULL, database=NULL){
+    DBI::dbConnect(RPostgreSQL::PostgreSQL(), host=host, user=username, password=password, dbname=database)
+}
+
+#' The metadata table. This is all of the different 
+#' codes that are available with their meanings, and
+#' what the different columns mean for them. 
```

Comment:
Is it worth adding to this what the structure of the table is, so callers know how to use it?

Reply:
I have given that a go